src/Disks/ObjectStorages/S3/ProxyResolverConfiguration.cpp

```diff
@@ -54,9 +54,6 @@ ClientConfigurationPerRequest ProxyResolverConfiguration::getConfiguration(const
         const auto & host = endpoint.getHost();
         auto resolved_hosts = DNSResolver::instance().resolveHostAll(host);
 
-        if (resolved_hosts.empty())
```

Comment:
We rely that resolveHostAll will throw exception if no hosts are resolved in a lot of places.